### PR TITLE
Json schema tests

### PR DESCRIPTION
**Pull request by seiferteric**

### CODE REVIEW

File: src/testdata/json_tests/interface_ipv4.json [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  ]
+}
```
Comment (jeff-yin) [inline src/testdata/json_tests/interface_ipv4.json]: Should you do a DELETE and GET at the end of this test to reset the interface config back to its original configuration?

File: src/testdata/json_tests/interface_mtu.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  ]
+}
```
Comment (jeff-yin) [inline src/testdata/json_tests/interface_mtu.json]: Same thing here, should reset MTU back to its default value when done

File: src/testdata/json_tests/interface_ipv4.json (hunk 1 above)
Comment (seiferteric) [reply src/testdata/json_tests/interface_ipv4.json]: Okay added cleanup operation.

File: src/testdata/json_tests/interface_mtu.json (hunk 2 above)
Comment (seiferteric) [reply src/testdata/json_tests/interface_mtu.json]: Added cleanup operation.

**Pull request merged**